HISTORY.rst

```diff
@@ -46,10 +46,15 @@ History
 * Added checkout parameter to branch creation and hard reset functions
 
 
- 0.2.4 (2021-01-11)
```

Comment:
Wow, that must have been quite the unexpected failure.  Maybe we could include a check in our (upcoming) github actions to verify syntax?

Reply:
Yes, that sounds like a wonderful idea, it was quite unexpected! I'll open an issue for that so it doesn't get forgotten.

Reply:
https://github.com/release-depot/git_wrapper/issues/75